src/ResourceManager/Resources/Commands.Resources/Models.ResourceGroups/ResourceClient.cs

```diff
@@ -378,13 +377,13 @@ public virtual List<GenericResourceExtended> FilterResources(FilterResourcesOpti
             return resources;
         }
 
-        public ProviderOperationsMetadata GetProviderOperationsMetadata(string providerNamespace)
+        public Management.Resources.Models.ProviderOperationsMetadata GetProviderOperationsMetadata(string providerNamespace)
```

Comment:
>Management.Resources.Models.ProviderOperationsMetadata [](start = 15, length = 54)

can you use this in the imports by giving an alias for this,as it is used in two different places